### PR TITLE
Delete doctor

### PR DESCRIPTION
**Pull request by kimshitong**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/NewCommandTestUtil.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+ * Contains helper methods for testing commands.
+ */
+public class NewCommandTestUtil {
+
+    public static final String VALID_NAME_AMY = "Amy Bee";
+    public static final String VALID_NAME_BOB = "Bob Choo";
+    public static final String VALID_NRIC_AMY = "T0123456A";
+    public static final String VALID_NRIC_BOB = "S9987654Z";
+    public static final String VALID_PHONE_AMY = "11111111";
+    public static final String VALID_PHONE_BOB = "22222222";
```
[the remark this answers is not in the document]
Comment (kimshitong) [reply src/test/java/seedu/address/logic/commands/NewCommandTestUtil.java:34]: Noted, I have changed this two.

Note with starting with 6,8,9

File: src/test/java/seedu/address/logic/commands/NewCommandTestUtil.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    public static final String VALID_EMAIL_AMY = "amy@example.com";
+    public static final String VALID_EMAIL_BOB = "bob@example.com";
+    public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
+    public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
+    public static final String VALID_TAG_HUSBAND = "husband";
+    public static final String VALID_TAG_FRIEND = "friend";
```
[the remark this answers is not in the document]
Comment (kimshitong) [reply src/test/java/seedu/address/logic/commands/NewCommandTestUtil.java:40]: Noted

File: src/test/java/seedu/address/testutil/TypicalDoctor.java [hunk 3]
```diff
@@ -65,4 +66,12 @@ public static List<Doctor> getTypicalDoctor() {
         return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE));
     }
 
+    public static Database getTypicalDatabase() {
+        Database ab = new Database();
```
[the remark this answers is not in the document]
Comment (kimshitong) [reply src/test/java/seedu/address/testutil/TypicalDoctor.java:70]: Ok

File: src/main/java/seedu/address/logic/commands/DeleteDoctorCommand.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.NewModel;
+import seedu.address.model.person.doctor.Doctor;
+
+/**
+ * Deletes a doctor identified using it's displayed index from the address book.
+ */
+public class DeleteDoctorCommand extends NewCommand {
+
+    public static final String COMMAND_WORD = "delete";
```
[the remark this answers is not in the document]
Comment (kimshitong) [reply src/main/java/seedu/address/logic/commands/DeleteDoctorCommand.java:20]: Ok, sorry overlooking

File: src/test/java/seedu/address/logic/commands/DeleteDoctorCommandTest.java [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+        Index targetIndex = Index.fromOneBased(1);
+        DeleteDoctorCommand deleteCommand = new DeleteDoctorCommand(targetIndex);
+
+        // same value -> returns same hashcode
+        assertEquals(deleteCommand.hashCode(), Objects.hash(targetIndex));
+
+        Index targetIndex2 = Index.fromOneBased(2);
+
+        // different feedbackToUser value -> returns different hashcode
+        assertNotEquals(deleteCommand.hashCode(), Objects.hash(targetIndex2));
```
[the remark this answers is not in the document]
Comment (kimshitong) [reply src/test/java/seedu/address/logic/commands/DeleteDoctorCommandTest.java:119]: understand.

**Review by derekjxtan — APPROVED** [review]
lgtm

**Pull request merged**